### PR TITLE
Refactor template-manager RPCs

### PR DESCRIPTION
**Pull request by BranislavMalo**
JIRA: UNIC-1285
TYPE: API

### CODE REVIEW

**Comment by BranislavMalo** [comment]
Reviewers:
@knuuzor
@jaro0149

**Review by knuuzor — APPROVED** [review]
Copy check OK

File: frinx-uniconfig/user-guide/uniconfig-operations/templates-manager/rpc_apply-template/readme.md [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+}'
+```
+
+```json RPC Response, Status: 500
+{
+  "errors": {
+    "error": [
+      {
+        "error-type": "application",
+        "error-tag": "operation-failed",
```
Comment (jaro0149) [inline frinx-uniconfig/user-guide/uniconfig-operations/templates-manager/rpc_apply-template/readme.md:233]: it should not be system error (500) - in this case, user did not provide value for some variable

File: frinx-uniconfig/user-guide/uniconfig-operations/templates-manager/rpc_create-multiple-templates/readme.md [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+}'
+```
+
+```json RPC Response, Status: 500
+{
+  "errors": {
+    "error": [
+      {
+        "error-type": "application",
+        "error-tag": "operation-failed",
```
Comment (jaro0149) [inline frinx-uniconfig/user-guide/uniconfig-operations/templates-manager/rpc_create-multiple-templates/readme.md:255]: this is user fault - user provided invalid data

**Review by jaro0149 — CHANGES_REQUESTED** [review]

File: frinx-uniconfig/user-guide/uniconfig-operations/templates-manager/rpc_create-multiple-templates/readme.md (hunk 2 above)
Comment (BranislavMalo) [reply frinx-uniconfig/user-guide/uniconfig-operations/templates-manager/rpc_create-multiple-templates/readme.md:255]: Done.

File: frinx-uniconfig/user-guide/uniconfig-operations/templates-manager/rpc_apply-template/readme.md (hunk 1 above)
Comment (BranislavMalo) [reply frinx-uniconfig/user-guide/uniconfig-operations/templates-manager/rpc_apply-template/readme.md:233]: Done.

File: frinx-uniconfig/user-guide/uniconfig-operations/templates-manager/rpc_create-multiple-templates/readme.md (hunk 2 above)
Comment (BranislavMalo) [reply frinx-uniconfig/user-guide/uniconfig-operations/templates-manager/rpc_create-multiple-templates/readme.md:255]: Done.

**Review by Vasyl-9 — APPROVED** [review]

**Review by jaro0149 — APPROVED** [review]

**Pull request merged**